pkgs/by-name/bo/bombsquad/package.nix

```diff
@@ -0,0 +1,96 @@
+{
+  stdenv,
```

Comment:
Are you using Alejandra?

Reply:
i dont even know what that is.

Reply:
[A code formatter for Nix](https://github.com/kamadorueda/alejandra).

One of its idiosyncrasies is to end each line of argset with `,`.
It is very ugly and goes against the typical style of Nixpkgs: 

```nix
{ lib
, arg1
, arg2
, arg3
}:
```